panther_manager/include/panther_manager/behavior_tree_utils.hpp

```diff
@@ -77,4 +77,19 @@ void RegisterBehaviorTree(
 
 }  // namespace panther_manager::behavior_tree_utils
 
+namespace panther_manager
+{
+// TODO: @pawelirh move to a separate file with an appropriate abstraction layer
```

Comment:
This is a different task I assume

Reply:
Yep, I left a comment to keep this tracked while reorganizing the BT architecture 👍🏼